thoughts/shared/plans/advisor-storage-architecture-fix.md

```
@@ -0,0 +1,517 @@
+                       {advisor : The advisor key}
+                       {--tag= : Export specific tagged version}
+                       {--job-id= : Export specific job ID}
+                       {--latest : Export latest completed version}';
+
+public function handle(): int
+{
+    $advisorKey = $this->argument('advisor');
+    
+    $job = null;
+    if ($this->option('tag')) {
+        $job = AdvisorGenerationJob::where('advisor_key', $advisorKey)
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)